docs/notebooks/feasible_sets.pct.py

```diff
@@ -230,6 +240,77 @@ def objective_function(x):
 )
 
 
+# %% [markdown]
+# ## Active learning with Integrated Variance Reduction acquisition function
+#
+# An alternative to the `ExpectedFeasibility` function is called `IntegratedVarianceReduction`. This acquisition has the advantage of allowing batches, as we illustrate below. It is more expensive to compute than `ExpectedFeasibility`, as it writes as an integral over a set of input locations.
+
+# %%
+from trieste.acquisition.function import IntegratedVarianceReduction
+
+# Choose integration points uniformly over the design space
+integration_points = search_space.sample_halton(1000)
+acq_ivr = IntegratedVarianceReduction(
+    integration_points=integration_points,
+    threshold=[
+        threshold,
+    ],
+)
+
+# Set a batch size greater than 1
+rule_ivr = EfficientGlobalOptimization(builder=acq_ivr, num_query_points=3)
+bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
+
+num_steps = 10
+result_ivr = bo.optimize(num_steps, initial_data, initial_model, rule_ivr)
+
+final_model_ivr = result_ivr.try_get_final_model()
+dataset_ivr = result_ivr.try_get_final_dataset()
+query_points_ivr = dataset_ivr.query_points.numpy()
+
+plot_excursion_probability(
+    "Final probability of excursion", final_model_ivr, query_points_ivr
+)
+
+# %% [markdown]
+# One can also specify a range of thresholds rather than a single value. The resulting query points are likely to be more speard out than previously.
```

Comment:
```suggestion
# One can also specify a range of thresholds rather than a single value. We can do this by specifying a range with a minimum and a maximum threshold, rather than a single threshold as the `threshold` parameter. The resulting query points are likely to be more spread out than previously, as now the whole region between the thresholds is aimed to be well estimated, rather than a single line.
```